src/api/ChatAPI.ts

```diff
@@ -0,0 +1,63 @@
+import { BaseAPI } from '../utils/BaseAPI';
+
+interface ChatFilters {
+    offset?: number;
+    limit?: number;
+    title?: string;
+}
+
+interface ChatData {
+    title?: string;
+}
+
+interface ChatUser {
+    users?: number [],
+    chatId?: number
+}
+
+interface ChatAvatar {
+    chatId?: number;
+    avatar?: FormData;
+}
```

Comment:
Стоит пробежаться по интерфейсам и привести оформление к одному виду
```suggestion
interface ChatUser {
    users?: number [];
    chatId?: number;
}

interface ChatAvatar {
    chatId?: number;
    avatar?: FormData;
}
```